data/mods/Magiclysm/monsters/monsters.json

```
@@ -539,5 +539,70 @@
     "death_function": { "corpse_type": "NO_CORPSE", "message": "The %s melts away." },
     "path_settings": { "avoid_traps": true, "avoid_sharp": true },
     "flags": [ "SEES", "WARM", "BASHES", "DESTROYS", "NO_BREATHE", "PUSH_MON", "PUSH_VEH", "FIREY", "PATH_AVOID_DANGER_1" ]
+  },
+  {
+    "id": "mon_razor_butterflies",
+    "type": "MONSTER",
+    "name": { "str": "razor butterfly", "str_pl": "razor butterflies" },
+    "//": "Now only for shaman, todo: make them natural spawn, add harvest, etc",
+    "description": "Some unknown species of butterfly, they have clear white wings with some orange rune pattern, and a razor-sharp part on the edge.  They are pretty big, but still in natural limits.",
```

i tried to do a dcss reference
idk if the dash should be there
```suggestion
    "description": "A large, brightly colored butterfly with crystalline wings, fluttering incongruously through the air.  This one seems to be patterned with orange runes, and its wings look razor-sharp.",
```